api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -39,6 +40,22 @@ message IPTagging {
     EXTERNAL = 2;
   }
 
+  // Describes how to apply the tags to the headers.
+  enum HeaderAction {
+    // (DEFAULT) The header specified in :ref:`ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`
+    // will be dropped, before the tags are applied. Sanitize will be run regardless of if the source is a trusted IP or not.
```

Comment:
```suggestion
    // will be dropped, before the tags are applied. The incoming header will be "sanitized" regardless of whether the request is an internal or external.
```

I'm trying to stick to the same terminology introduced in https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/header_sanitizing. I'm assuming that is the goal, and if so, I suggest trying to stick with the same terms.